Content.Shared/_Goobstation/Bingle/BinglePitFallingComponent.cs

```diff
@@ -0,0 +1,40 @@
+using System.Numerics;
+using Robust.Shared.GameStates;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom;
+
+namespace Content.Shared._Goobstation.Bingle;
+
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentPause]
+public sealed partial class BinglePitFallingComponent : Component
+{
+    /// <summary>
+    ///     Time it should take for the falling animation (scaling down) to complete.
+    /// </summary>
+    [DataField("animationTime")]
+    public TimeSpan AnimationTime = TimeSpan.FromSeconds(1.5f);
+
+    /// <summary>
+    ///     Time it should take in seconds for the entity to actually delete
+    /// </summary>
+    [DataField("deletionTime")]
+    public TimeSpan DeletionTime = TimeSpan.FromSeconds(1.8f);
+
+    [DataField("nextDeletionTime", customTypeSerializer:typeof(TimeOffsetSerializer))]
+    [AutoPausedField]
+    public TimeSpan NextDeletionTime = TimeSpan.Zero;
+
+    /// <summary>
+    ///     Original scale of the object so it can be restored if the component is removed in the middle of the animation
+    /// </summary>
+    public Vector2 OriginalScale = Vector2.Zero;
+
+    /// <summary>
+    ///     Scale that the animation should bring entities to.
+    /// </summary>
+    public Vector2 AnimationScale = new Vector2(0.01f, 0.01f);
+
+    /// <summary>
+    ///     the pit your about to fall into
+    /// </summary>
+    public BinglePitComponent Pit = new BinglePitComponent();
```

Comment:
```suggestion
    public BinglePitComponent? Pit;
```
This isn't resolved, we really can't instantiate components here, doubly so if we never even use the value anyway.
After applying this fix, all you need to do is a null-check on places where the Pit is used (because we're marking it as potentially null), and then it'll work just like it did before.

Reply:
dont know hwo to do that